### ISSUE TITLE
`--version` flag returns extra error message

### ISSUE THREAD

**Issue by sharpchen** [fsprojects/FSharpLint#741]
## Repro

```sh
$ dotnet-fsharplint --version

Current version: 0.24.2+143ae50e0e32d0842fa36c1bc22c735f2c72f114
no valid subcommand has been specified.

SUBCOMMANDS:

    lint <options>        Runs FSharpLint against a file or a collection of
                          files.

    Use 'fsharplint <subcommand> --help' for additional information.

OPTIONS:

    --format, -f <standard|msbuild>
                          Output format of the linter.
    --version             Prints current version.
    --help                display this list of options.
```
